src/sentry/static/sentry/app/components/dropdownControl.tsx

```diff
@@ -71,12 +88,12 @@ class DropdownControl extends React.Component {
               {this.renderButton(isOpen, getActorProps)}
               <Content
                 {...getMenuProps()}
-                alignMenu={alignRight ? 'right' : 'left'}
+                alignMenu={alignMenu}
                 width={menuWidth}
-                menuOffset={menuOffset}
                 isOpen={isOpen}
                 blendCorner
                 blendWithActor={blendWithActor}
+                theme={theme}
```

Comment:
Hmm was this giving you errors? You shouldn't need to pass `theme` like this, when you use `styled()` it injects `theme` as props.